ide/jumpto/src/org/netbeans/modules/jumpto/file/FileSearchAction.java

```diff
@@ -220,6 +220,7 @@ public boolean setListModel(final FileSearchPanel panel, String text ) {
                 itemsComparator.setUsePreferred(panel.isPreferedProject());
                 itemsComparator.setText(text);
                 filterFactory.setLineNumber(lineNr);
+                //TODO: EMI: searchByFolder?
```

Comment:
Does this mean anything? If not, please remove, otherwise it will be hard later to figure out again what that means :).